File: app/src/main/java/io/github/gusandrianos/foxforreddit/data/models/Thing.java
package io.github.gusandrianos.foxforreddit.data.models;

import com.google.gson.annotations.SerializedName;

public class Thing {

    @SerializedName("data")
    private Data data;

    @SerializedName("kind")
    private String kind;

    public Data getData() {
        return data;
    }

    public String getKind() {
        return kind;
    }
}

File: app/src/main/java/io/github/gusandrianos/foxforreddit/data/models/Thing.kt
package io.github.gusandrianos.foxforreddit.data.models

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
class Thing : Parcelable{
    @SerializedName("data")
    val data: Data? = null

    @SerializedName("kind")
    val kind: String? = null
}